distribution/docs/src/main/resources/content/_metadataAttributes/nsili-attributes-table.adoc

```diff
@@ -304,7 +304,7 @@ else, `id`
 |cloudCoverPercentage
 |`isr.cloud-cover`
 |none
-|Short
+|Double
```

Comment:
Does the ISR attribute need to be updated too?
https://github.com/codice/alliance/blob/d559287819156c69ff451b45b67e78f8ac7a94f1/catalog/core/catalog-core-api-impl/src/main/java/org/codice/alliance/catalog/core/api/impl/types/IsrAttributes.java#L32-L39